W-10673838 veeva vault clean up (#1802)
mulesoft/docs-connectors: veevavault/1.3/modules/ROOT/pages/veevavault-connector-xml-maven.adoc

= Veeva Vault Connector 1.3 - XML and Maven Support - Mule 4
:page-aliases: connectors::veevavault/veevavault-connector-xml-maven.adoc

Although you can manually code a Mule app in XML, it is more efficient to use Anypoint Studio:

* If you manually code a Mule runtime engine (Mule) app in XML (either from the Anypoint Studio XML editor or from a text editor), you can access the connector from your app by adding reference statements to both your XML Mule flow and the Apache Maven `pom.xml` file.

> * If you add the connector in Studio, Studio automatically populates the XML code with the connector's namespace and schema location and it also adds a dependency to the `pom.xml` file.

Reviewer: ```suggestion
* If you add the connector in Anypoint Studio, Studio automatically populates the XML code with the connector's namespace and schema location and adds a dependency to the `pom.xml` file.
```

</dependency>
----

> Replace `<version>` with the version that corresponds to the connector you are using.

Reviewer: Is this supposed to name Veeva Vault Connector rather than "the connector you are using"?

To obtain the most up-to-date `pom.xml` file information:

. Go to https://www.mulesoft.com/exchange/[Anypoint Exchange].
. In Exchange, click *Login* and supply your Anypoint Platform username and password.
. In Exchange, search for `connector-template-files`.
. Select the connector.

> . Click *Dependency Snippets* near the upper right of the screen.

Reviewer: Just a note: This kind of location language is going to have to be revised for accessibility: near the upper right of the screen. We need to globally work with UX to be sure that users can locate controls by labels alone.

Author: Good point! Will revisit this as we are planning on revamping our docs for accessibility
